OpenRA.Mods.Common/LoadScreens/LogoStripeLoadScreen.cs

```diff
@@ -79,12 +79,15 @@ public override void Display()
 			r.EndFrame(new NullInputHandler());
 		}
 
-		public override void Dispose()
+		protected override void Dispose(bool disposing)
 		{
-			if (sheet != null)
-				sheet.Dispose();
+			if (disposing)
```

Comment:
I tend to format it like this so future people can just dunk any extra stuff into the section without redoing the whole block, e.g.:
```
if (disposing)
{
    if (a != null)
        a.Dispose();
    b.Dispose();
}